openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -78,15 +78,15 @@ const Carousel = {
             borrow_available: {cls: 'cta-btn--available', cta: 'Borrow'},
             borrow_unavailable: {cls: 'cta-btn--unavailable', cta: 'Join Waitlist'},
             error: {cls: 'cta-btn--missing', cta: 'Not In Library'},
-            private: {cls: 'cta-btn--available', cta: 'Preview'}
+            // private: {cls: 'cta-btn--available', cta: 'Preview'}
```

Comment:
Is this intentionally commented out?  Oddly, I'm still seeing "Preview" as a call to action for some carousel items that have been added after advancing the carousel:

![carousel-preview-cta](https://user-images.githubusercontent.com/28732543/125118564-e4fa9400-e0bd-11eb-950f-9beee8e69402.png)

Reply:
Let me try re-PR deploying; maybe I missed the last commit.

Reply:
I commented it out because it's very different from our normal loan status buttons? We've got to find a way to DRY up this code :/

Reply:
I think I get it.  You're talking about the button's behavior (opening a preview modal as opposed to redirecting like the other buttons do)?

Reply:
Previewable books in carousels now appear as "Not in library", as expected with these changes.  Opened issue #5407 to improve loan status button consistency across the site.